engine/src/srvcgi.cpp

```diff
@@ -707,12 +707,11 @@ static void cgi_store_form_urlencoded(MCVariable *p_variable, MCDataRef p_data,
 
 static void cgi_fix_path_variables()
 {
-	char *t_path, *t_path_end;
+    char *t_path;
 
 	MCStringRef env;
     env = nil;
-	t_path = nil;
-    t_path_end = nil;
+    t_path = nil;
     
     // SN-2014-07-29: [[ Bug 12865 ]] When a LiveCode CGI script has a .cgi extension and has
```

Comment:
That's simply for a backwards-compatibility that this was reinstated, as this has always been done by the engine itself, not Apache (or such).
As Apache doc seems to say (http://httpd.apache.org/docs/2.2/mod/core.html#acceptpathinfo), I'm not sure that the PATH_INFO must be set - but more that extra segments are allowed. Even with the AcceptPathInfo explicitly set to 'on', the URL described in the bug report is not working with LC 7.0.1